modal/partial_function.py

```diff
@@ -44,8 +44,8 @@ def all() -> int:
 R = typing_extensions.TypeVar("R", covariant=True)
 
 
-class _PartialFunction(typing.Generic[P, T]):
-    """Intermediate function, produced by @enter, @build, @method, @web_endpoint, or @batched"""
+class _PartialFunction(typing.Generic[P, R]):
+    """Intermediate function, produced by @enter, @build, @method, @web_endpoint or @batched"""
```

Comment:
Clear your schedule for the rest of the day we need to bikeshed about oxford commas :D

Reply:
😰